api/types/accesslist/accesslist.go

```diff
@@ -279,14 +245,14 @@ func NewAccessList(metadata header.Metadata, spec Spec) (*AccessList, error) {
 // set. Any other invalid value is an error.
 func checkInclusion(i Inclusion) (Inclusion, error) {
 	switch i {
-	case "":
-		return Explicit, nil
+	case InclusionUnspecified:
+		return InclusionExplicit, nil
 
-	case Explicit, Implicit:
+	case InclusionExplicit, InclusionImplicit:
 		return i, nil
 
 	default:
-		return Explicit, trace.BadParameter("invalid inclusion mode %q (must be %q or %q)", i, Explicit, Implicit)
+		return InclusionUnspecified, trace.BadParameter("invalid inclusion mode %d (must be %d or %d)", i, InclusionExplicit, InclusionImplicit)
```

Comment:
Maybe: Use the string values in the messages, apart from the unknown one?

```suggestion
		return InclusionUnspecified, trace.BadParameter(
			"invalid inclusion mode %d (must be %q/%d or %q/%d)",
			i, InclusionExplicit, InclusionExplicit, InclusionImplicit, InclusionImplicit)
```